diff --git a/Source/src/commands.cpp b/Source/src/commands.cpp
index 6c4fbb1..c2b1ae9 100644
--- a/Source/src/commands.cpp
+++ b/Source/src/commands.cpp
@@ -2361,19 +2361,7 @@ void commandMicrostep()
   }
   
   printf(str_MicrostepDescription);
-  
-  // too bad, we're out of pins here that can sink 48 mA :)
-  if (hdd.getParams()->UseReduceWriteCurrent)
-  {
-    printf(str_MicrostepRWCInUse);
-    return;
-  }
-  if (hdd.getParams()->Heads > 8)
-  {
-    printf(str_MicrostepTooManyHeads, hdd.getParams()->Heads);
-    return;
-  }
-  
+
   // test recovery mode if it works
   printf(str_MicrostepTesting);
   const bool test = hdd.testMicrostepping();
diff --git a/Source/src/hdd.cpp b/Source/src/hdd.cpp
index e9ad3f2..f18e050 100644
--- a/Source/src/hdd.cpp
+++ b/Source/src/hdd.cpp
@@ -13,6 +13,7 @@ HDD::HDD()
   m_PhysicalCylinder = 0;
   m_PhysicalHead = 0;
   
+  m_RecoveryMode = false;
   m_MicroStepOnNoAddressMark = false;
   m_MicroStepOnCRCError = false;
   
@@ -294,11 +295,11 @@ bool HDD::seekDrive(uint16_t toCylinder, uint8_t toHead)
   return true;
 }
 
-// reusing the HDSEL3 line wired up to the RECOVERY MODE signal of the hard drive
+// GPIO1 connected to a 1K resistor to the base of a BJT NPN; emitter to ground, collector to /RECOVERYMODE (usually pin 16 of disk control cable)
 void HDD::microStep(bool perform)
 {
   const bool isEnabled = m_MicroStepOnNoAddressMark || m_MicroStepOnCRCError;
-  if (!isEnabled || (m_Params.Heads > 8) || m_Params.UseReduceWriteCurrent)
+  if (!isEnabled)
   {
     return;
   }
@@ -344,19 +345,19 @@ bool HDD::testMicrostepping()
 bool HDD::microStepInternal(bool perform, bool testing)
 {
   const bool isEnabled = testing || m_MicroStepOnNoAddressMark || m_MicroStepOnCRCError;
-  if (!isEnabled || (m_Params.Heads > 8) || m_Params.UseReduceWriteCurrent)
+  if (!isEnabled)
   {
     return false;
   }  
   
   if (!perform)
   {
-    if (m_ShiftRegister & 8)
+    if (m_RecoveryMode)
     {
       // leave recovery mode
       g_SeekComplete = false;
-      m_ShiftRegister &= 0xF7;
-      updateShiftRegister();
+      m_RecoveryMode = false;
+      gpio_put(1, m_RecoveryMode);
       sleep_ms(SLOWSEEK_SRT_MS);
       
       // disk must assert seek complete
@@ -379,10 +380,10 @@ bool HDD::microStepInternal(bool perform, bool testing)
   }
    
   // enter recovery mode if not already in it
-  if (!(m_ShiftRegister & 8))
+  if (!m_RecoveryMode)
   {
-    m_ShiftRegister |= 8;
-    updateShiftRegister();
+    m_RecoveryMode = true;
+    gpio_put(1, m_RecoveryMode);
     sleep_us(1); 
   }
   
diff --git a/Source/src/hdd.h b/Source/src/hdd.h
index 59ac8f4..9a8a45d 100644
--- a/Source/src/hdd.h
+++ b/Source/src/hdd.h
@@ -78,6 +78,7 @@ private:
   uint16_t m_PhysicalCylinder;
   uint8_t m_PhysicalHead;
   
+  bool m_RecoveryMode;
   bool m_MicroStepOnNoAddressMark;
   bool m_MicroStepOnCRCError;
 };
\ No newline at end of file
diff --git a/Source/src/main.cpp b/Source/src/main.cpp
index 6421e2f..0f54120 100644
--- a/Source/src/main.cpp
+++ b/Source/src/main.cpp
@@ -175,20 +175,21 @@ int main()
   
   srand(get_rand_32());
    
-  // GPIOs used: 2-22, 26-28
-  gpio_init_mask(0x1C7FFFFC);
+  // GPIOs used: 1-22, 26-28
+  gpio_init_mask(0x1C7FFFFE);
 
   // set initial states of GPIOs
   // outputs:
   // 28 (separator: SC), 27 (74HC595: SER), 26 (74HC595: /OE), 22 (74HC595: RCLK), 21 (74HC595: SRCLK), 12 (separator: WPCEN), 
-  // 11 (separator: /EARLY), 10 (separator: /LATE), 8 (separator: WDATA), 7 (separator: WGATE), 3 (separator: RGATE)
+  // 11 (separator: /EARLY), 10 (separator: /LATE), 8 (separator: WDATA), 7 (separator: WGATE), 3 (separator: RGATE),
+  // 1 (drive: RECOVERYMODE through a transistor)
   // inputs:
   // 20 (drive: /WFAULT), 19 (SD: MOSI), 18 (SD: SCK), 17 (SD: SCS), 16 (SD: MISO), 15 (drive: /READY), 14 (drive: /TRK0),
   // 13 (drive: /SC), 9 (separator: WCLOCK), 6 (drive: /INDEX), 5 (separator: RCLOCK), 4 (separator: RDATA), 2 (separator: DRUN)
-  gpio_set_dir_masked(0x1C7FFFFC, 0x1C601D88);
+  gpio_set_dir_masked(0x1C7FFFFE, 0x1C601D8A);
 
   // drive outputs: 26 (74HC595: /OE), 11 (separator: /EARLY), 10 (separator: /LATE) high; and the rest of the outputs low
-  gpio_put_masked(0x1C601D88, 0x4000C00);
+  gpio_put_masked(0x1C601D8A, 0x4000C00);
 
   // set initial states of the output shift register 74HC595
   // (Q0: HDSEL0, Q1: HDSEL1, Q2: HDSEL2, Q3: HDSEL3, Q4: STEP, Q5: DIR, Q6: DS, Q7: RLL/MFM)
diff --git a/Source/src/stringtable.h b/Source/src/stringtable.h
index 1758af0..bd92858 100644
--- a/Source/src/stringtable.h
+++ b/Source/src/stringtable.h
@@ -226,9 +226,9 @@ STRINGTABLE str_WriteDetailsFormat[]      = "\nWill format %u cylinders.\n";
 STRINGTABLE str_WriteSeagateCylinder[]    = "Updating Seagate controller-reserved cylinder 0...";
 
 // microstepping command
-STRINGTABLE str_MicrostepDescription[]    = "\nReuses disk control cable pin 2 (/HDSEL3) to enable /RECOVERYMODE.\n"
-                                            "Use this to mitigate disk data recording that has drifted over time:\n"
-                                            "wire /HDSEL3 to the recovery mode signal of the disk connector.\n"
+STRINGTABLE str_MicrostepDescription[]    = "\nUse this to mitigate disk data recording that has drifted over time:\n"
+                                            "wire GPIO1 to a BJT NPN through a 1K resistor to base, emitter to GND,\n"
+                                            "collector to the recovery mode signal of the disk control cable.\n"
                                             "Most likely, a jumper will need to be set for proper operation.\n";
 STRINGTABLE str_MicrostepOnNoAddrMark[]   = "Microstepping on sector not found / no data address mark errors? Y/N: ";
 STRINGTABLE str_MicrostepOnCRCErrors[]    = "Microstepping on uncorrectable CRC errors? Y/N: ";
@@ -236,8 +236,6 @@ STRINGTABLE str_MicrostepTesting[]        = "\nTesting recovery mode...";
 STRINGTABLE str_MicrostepTestFail[]       = "\nDrive not microstepping; recovery mode not enabled.\n";
 STRINGTABLE str_MicrostepApplied[]        = "Microstepping on. Up to %u algorithms will be tested before reporting errors.\n";
 STRINGTABLE str_MicrostepReseekOff[]      = "NOTE: re-seeking after sector errors turned off until board reset.\n";
-STRINGTABLE str_MicrostepRWCInUse[]       = "\nCannot reuse /HDSEL3 if reduced write current signal is used.\n";
-STRINGTABLE str_MicrostepTooManyHeads[]   = "\nCannot reuse /HDSEL3; %u disk drive heads configured.\n";
 STRINGTABLE str_MicrostepOff[]            = "OFF";
 STRINGTABLE str_MicrostepOn[]             = "ON";
 
